easybuild/easyblocks/n/ncl.py

```diff
@@ -78,15 +78,19 @@ def configure_step(self):
         ctof_libs = ''
         ifort = get_software_root('ifort')
         if ifort:
-            if LooseVersion(get_software_version('ifort')) < LooseVersion('2011.4'):
-                ctof_libs = '-lm -L%s/lib/intel64 -lifcore -lifport' % ifort
+            if os.path.exists('%s/lib/intel64/' % ifort) and os.access('%s/lib/intel64/' % ifort, os.R_OK):
+                ctof_libs += '-lm -L%s/lib/intel64/ -lifcore -lifport' % ifort
             else:
-                ctof_libs = '-lm -L%s/compiler/lib/intel64 -lifcore -lifport' % ifort
+                self.log.warning(
+                    "Can't find a libdir for ifortran libraries -lifcore -lifport: "
+                    "%s/lib/intel64 doesn't exist or is not accessible." % (ifort, ifort)
```

Comment:
One too many ifort in the args list there

Reply:
Well spotted, I probably had a different phrase earlier on... corrected it now :)